instructor/common.py

```diff
@@ -1334,6 +1335,25 @@ def create_image_mappings(assignment, sample):
 def microscopy_analyze(request):
     pk = request.session['assignment_id']
     assignment = models.Assignment.objects.get(id=pk)
+    chosen_sampleprep = ""
+    chosen_protocol = ""
+    mapping_pk = ""
+    dialog_open = False
+    if request.method == "POST" and 'continue' in request.POST:
+        return redirect('common_assignments')
+    if request.method == "POST" and 'upload' in request.POST:
+        if len(request.FILES) > 0:
+            uploaded_file = request.FILES['file']
+
+            new_image = models.MicroscopyImage(file=uploaded_file)
```

Comment:
no reason, I'll change to `get_or_create` to avoid upload of the same image more than once.